service_invocation/csharp/http/checkout/Program.cs

```diff
@@ -1,24 +1,21 @@
-﻿using System.Text;
-using System.Text.Json;
-using System.Text.Json.Serialization;
+﻿using Dapr.Client;
 
-var baseURL = (Environment.GetEnvironmentVariable("BASE_URL") ?? "http://localhost") + ":" + (Environment.GetEnvironmentVariable("DAPR_HTTP_PORT") ?? "3500");
+using var client = new DaprClientBuilder().Build();
 
-var client = new HttpClient();
-client.DefaultRequestHeaders.Accept.Add(new System.Net.Http.Headers.MediaTypeWithQualityHeaderValue("application/json"));
-// Adding app id as part of the header
-client.DefaultRequestHeaders.Add("dapr-app-id", "order-processor");
-
-for (int i = 1; i <= 20; i++) {
-    var order = new Order(i);
-    var orderJson = JsonSerializer.Serialize<Order>(order);
-    var content = new StringContent(orderJson, Encoding.UTF8, "application/json");
+for (int i = 1; i <= 20; i++)
+{
+    var data = new { orderId = i };
 
+    var cts = new CancellationTokenSource();
+    Console.CancelKeyPress += (object? sender, ConsoleCancelEventArgs e) => cts.Cancel();
     // Invoking a service
-    var response = await client.PostAsync($"{baseURL}/orders", content);
-    Console.WriteLine("Order passed: " + order);
+    await client.InvokeMethodAsync<object>(

```

Comment:
This is an example of what we don't want to change.  This changes from using .NET's HttpClient type, to using the DaprClient SDK type.  The line of code I show you above hands back an HttpClient, and the rest of the code that relied on HTTPclient should be unchanged.

Reply:
Got it, thanks.